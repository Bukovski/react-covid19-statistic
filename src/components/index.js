export { default as Chart } from './chart/chart.component';
export { default as Info } from './info/info.component';
export { default as Countries } from './countries/countries.component';
