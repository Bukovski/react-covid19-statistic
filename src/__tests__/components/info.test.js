import React from 'react';
import { render, screen } from "@testing-library/react";

import { Info } from "../../components";
import { mockStatistic } from "../../__mocks__/axios-statistic.mock";
import expect from "expect";



describe.only("Info", () => {
	it("render page with props", () => {
		const { container, queryAllByText } = render(<Info data={ mockStatistic }/>);
		
		expect(queryAllByText(/3 ноября 2021/)).toBeTruthy();
		expect(container.firstChild).toMatchSnapshot();
	})
})