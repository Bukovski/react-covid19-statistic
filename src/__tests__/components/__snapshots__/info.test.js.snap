// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info render page with props 1`] = `
<div
  class="info-container"
>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      class="MuiPaper-root MuiCard-root MuiGrid-root card infected MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
        />
        <h2
          class="MuiTypography-root MuiTypography-h5"
        >
          <span>
            0
          </span>
        </h2>
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          3 ноября 2021 г.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        />
      </div>
    </div>
    <div
      class="MuiPaper-root MuiCard-root MuiGrid-root card recovered MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
        />
        <h2
          class="MuiTypography-root MuiTypography-h5"
        >
          <span>
            0
          </span>
        </h2>
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          3 ноября 2021 г.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        />
      </div>
    </div>
    <div
      class="MuiPaper-root MuiCard-root MuiGrid-root card deaths MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
        />
        <h2
          class="MuiTypography-root MuiTypography-h5"
        >
          <span>
            0
          </span>
        </h2>
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          3 ноября 2021 г.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        />
      </div>
    </div>
  </div>
</div>
`;
